# the buoyant center pos relative to the origin of the robot coordinate frame
Buoyant_Force : 0
Buoyancy_X_POS : 0
Buoyancy_Y_POS : 0
Buoyancy_Z_POS : 0
IS_Submerged : false